bot/discord/commands/help.js

```diff
@@ -1,52 +1,26 @@
 const Discord = require('discord.js')
-const commands = {
-    Users: {
-        user: "See help for that command.",
-        server: "See help for that command.",
-        stats: "Shows the stats of each hosting node.",
-        ping: "Shows the bots ping.",
-        ticket: "Create a ticket for help from the staff team!.",
-        uptime: "Shows the bots uptime.",
-        vc: "Manage you custom voice channel.",
-        info: "Get a bots info.",
-        animalapi: "Check out our animal api."
-    },
-    Staff: {
-        staff: "See help for that command.",
-        purge: "Delete messages in a channel.",
-        mute: "Mute da user .",
-        kick: "Kick da user."
-    },
-    Owner: {
-        reload: "Reloads all commands on the bot.",
-        'staff update': "Pull latest git commit.",
-        eval: "Eval some code.",
-        exec: "Run some system commands.",
-        giveaway: "Launch a giveaway.",
-        announce: "Announce something."
-    }
-}
-
-let desc = (object) => {
-    let description = [];
-    let entries = Object.entries(object);
-    for (const [command, desc] of entries) {
-        description.push(`**${config.DiscordBot.Prefix}${command}** - ${desc}`)
-    }
-    return description;
-}
 
 exports.run = async (client, message, args) => {
 
+    let prefix = 'DBH!' // config.DiscordBot.Prefix
+
     let embed = new Discord.MessageEmbed()
-        .setColor('BLUE')
-        .addField(`__**Users:**__ (${Object.entries(commands.Users).length})`, desc(commands.Users).join('\n'))
+    .setAuthor(`${client.user.username} | Help`, client.user.avatarURL())
+    .addField(`📺 | Help Section`, `> \`${prefix}user help\`\n> \`${prefix}help server\`\n> \`${prefix}ticket help\``)
+    .setThumbnail(client.user.avatarURL())
+    .setColor(message.guild.me.displayHexColor)
+    .setTimestamp()
 
-    if (message.member.roles.cache.get('748117822370086932') != null)
-        embed.addField(`__**Staff Commands:**__ (${Object.entries(commands.Staff).length})`, desc(commands.Staff).join('\n'))
+    if (message.member.roles.cache.get('748117822370086932')) {
+        embed.addField(`💡 | Staff Help`, `> \`${prefix}staff help\``)
+    }
 
-    if (message.member.roles.cache.find(r => r.id === "778237595477606440"))
-        embed.addField(`__**Developer Commands:**__ (${Object.entries(commands.Owner).length})`, desc(commands.Owner).join('\n'))
+    embed.addField(`❓ | Mics Commands`, `> \`${prefix}stats\`\n> \`${prefix}ping\`\n> \`${prefix}uptime\`\n> \`${prefix}vc\`\n> \`${prefix}info\``, true)
 
-    message.channel.send(embed)
-};
+    if (message.member.roles.cache.get('778237595477606440')) {
+        embed.addField(`👑 | Owner Commands`, `> \`${prefix}reload\`\n> \`${prefix}eval\`\n> \`${prefix}exec\`\n> \`${prefix}reload\`\n> \`${prefix}giveaway\`\n> \`${prefix}announce\``, true)
+    }
+
+    await message.channel.send({ embed: embed })
```

Comment:
you dont need to do `embed: embed` here, es6 does it for you and this is a much needed pr, its way better than the hardcoded one. Good Job!

Reply:
I was doing it for v13 so it would be nice for v13 :D

Reply:
yeah no im saying you dont need to do `{ embed: embed }`, you can just do `{ embed }` because the property name and the variable name is the same.

Reply:
Or just do: `message.channel.send(embed)` lol

Reply:
> I was doing it for v13 so it would be nice for v13 :D

it wouldn't work for v13 as you send embeds like message.channel.send({embeds: [embed]})